app/seeders/admin_user_seeder.rb

```diff
@@ -27,14 +27,10 @@
 #++
 class AdminUserSeeder < Seeder
   def seed_data!
-    user = new_admin
-    if user.save!(validate: false)
-      seed_data.store_reference(:openproject_admin, user)
+    if Setting.seed_admin_user_enabled?
+      seed_admin!
     else
-      print_error "Seeding admin failed:"
-      user.errors.full_messages.each do |msg|
-        print_error "  #{msg}"
-      end
+      Seeder.logger.debug { "   *** skipped as explicity disabled" }
```

Comment:
```suggestion
      Seeder.logger.debug { "   *** skipped as explicity disabled with OPENPROJECT_SEED_ADMIN_USER_ENABLED" }
```

Just in case the admin seeding is disabled and the user reading logs has no idea why.